dataconf/main.py

```diff
@@ -60,31 +70,35 @@ def on(self, clazz):
 multi = Multi([])
 
 
-def env(prefix: str, clazz, **kwargs):
+def env(prefix: str, clazz: Type, **kwargs):
     return multi.env(prefix, **kwargs).on(clazz)
```

Comment:
What is Type class? I am unfamiliar. Will this be a config tree?

Reply:
This is the [type](https://docs.python.org/3/library/typing.html#typing.Type) of types.